src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx

```diff
@@ -24,11 +24,125 @@ redirects:
   - /docs/logs/enable-log-management-new-relic/configure-logs-context/net-configure-logs-context-all
 ---
 
-Logs in context for the .NET agent connects your logs and APM data in New Relic. Bringing all of this data together in a single tool helps you quickly get to the root cause of an issue and find the log lines that you need to identify and resolve a problem.
+APM logs in context connects your logs with all of your telemetry data for your apps, hosts, and other entities. Bringing all of this data together in a single tool helps you quickly:
 
-## Set up your .NET app [#enable-logs-net]
+* Cut through the noise of thousands of logs when troubleshooting time-critical issues, so you automatically see only the most relevant logs.
+* Navigate within multiple types of telemetry data, and have the data correlate back to the original issue.
+* Easily drill down into more detailed information from the same place in the UI.
+* Find the log lines that you need to identify and resolve a problem.
 
-To enable logs in context for APM apps monitored by .NET:
+
+![APM logs in context](./images/ruby-errors-log-details.png "APM logs in context")
+
+<figcaption>
+  **APM logs in context:** From the left nav in the APM UI, you can select an error and immediately see related logs, log details, host name, important attributes, and more.
+</figcaption>
+
+For more information, including examples, learn how to get started with [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context).
+
+## Automatic logs in context options [#automatic]
+
+You have three options to configure APM logs in context to send your app's logs and linking metadata automatically to New Relic.
+
+<CollapserGroup>
+
+  <Collapser
+    id="1-agent"
+    title="Option 1 (simplest): Let the agent decorate and forward your logs."
+  >
+
+  This is the simplest approach, and it's a great choice for developers who may not have the access or interest in setting up a log forwarder, or for accounts that want to see the power of logs and other linking metadata in context of their apps, without a lot of overhead.
+  
+  Using this option, your logs will include `span.id`, `trace.id`, `hostname`, `entity.guid`, and `entity.name`. These attributes automatically link your logs to spans, traces, and other telemetry, making it easier to troubleshoot.
+
+  All you need to do is install an agent version with log forwarding capabilities ([.NET agent 9.7.0 or higher](/docs/release-notes/agent-release-notes/net-release-notes)), and update your configuration to enable forwarding.
+
+  **INSERT .NET-specific configuration**
```

Comment:
This block is the simplest form of enabling log forwarding using the newrelic.config file.

```xml
<applicationLogging enabled="true">
 <logForwarding enabled="true/>
</applicationLogging>
```